src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Helpers/Conversions/JobConversions.cs

```diff
@@ -152,8 +160,41 @@ private static AzureRmRecoveryServicesAzureVmJobErrorInfo GetPSAzureVmErrorInfo(
             return psErrorInfo;
         }
 
+        private static string GetJobIdFromFullId(string fullId)
+        {
+            string[] splitArr = fullId.Split("/".ToCharArray());
+            return splitArr[splitArr.Length - 1];
+        }
+
+        #endregion
+
         #endregion
 
+        #region Enum translators
+
+        public static string GetJobTypeForService(BackupManagementType mgmtType)
```

Comment:
This is specific to jobs. I am not sure if others want to use this. I'll keep it in this file.